workflow/environments/README.md

```diff
@@ -20,9 +20,10 @@ Requirements: Setup github SSH keys for maui,
 
 A user specific environment of workflow, qcore, IMCalc, Empirical Engine and Pre-processing 
 can be created as follows:  
-1) Go to an existing workflow repository
-2) Navigate to ".../slurm_gm_workflow/workflow/environments/org/nesi" 
-3) Run 
+1) Go to an existing copy of workflow (eg. can be your home) - it is a good idea to update this copy
+2) Navigate to ".../slurm_gm_workflow/workflow/environments/org/nesi"
+3) Load the correct version of Python module. (** IMPORTANT **)
```

Comment:
Do we want to state what the "correct" version of Python module is? like how it's done in step 8?

Reply:
Good suggestion!